app/src/main/res/layout-land/profile_reset_pin_activity.xml

```diff
@@ -101,6 +103,7 @@
             android:layout_marginStart="168dp"
             android:layout_marginTop="16dp"
             android:layout_marginEnd="168dp"
+            android:textDirection="locale"
```

Comment:
ditto here

Reply:
@ShridharGoel Please reply to these comments.

Reply:
This has been updated.